money-diary/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../reset.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Synthia's Money Diary</title>
+  </head>
+  <body>
+    <header>
+      <h1>Money Diary💵</h1>
+    </header>
+    <main>
+      <section class="balance">
+        <article class="balance">
+          <h2>My Balance</h2>
+          <h3>346000</h3>
+          <article class="detail">
+            <h5 class="deposit">➕ 300000</h5>
+            <h5 class="withdraw">➖ 154000</h5>
+          </article>
+        </article>
+      </section>
+      <section class="list">
+        <span class="date">
+          <button type="button"><</button>
+          <time> 10.10 </time>
+          <button type="button">></button>
+        </span>
+        <span class="checkbox">
+          <h3>History</h3>
+          <span class="buttons">
+            <button class="check_button" type="button">
+              <label>
+                <input type="checkbox" checked />
+                <span> Deposit</span>
+              </label>
+            </button>
+            <button class="check_button" type="button">
+              <label>
+                <input type="checkbox" checked />
+                <span> Withdraw</span>
+              </label>
+            </button>
+          </span>
+        </span>
+        <section class="history">
+          <article class="each_item">
+            <span>
+              <p class="category">Entertainment</p>
+              <p class="title">Bowling</p>
+            </span>
+            <span>
+              <p class="price_withdraw">-12000</p>
+              <button class="delete_button">X</button>
+            </span>
+          </article>
+          <article class="each_item">
+            <span>
+              <p class="category">Groceries</p>
+              <p class="title">GS25</p>
+            </span>
+            <span>
+              <p class="price_withdraw">-3000</p>
+              <button class="delete_button">X</button>
+            </span>
+          </article>
```

Comment:
```suggestion
        <section class="history">
        <ul>
          <li class="each_item">
            <span>
              <p class="category">Entertainment</p>
              <p class="title">Bowling</p>
            </span>
            <span>
              <p class="price_withdraw">-12000</p>
              <button class="delete_button">X</button>
            </span>
          </li>
          <li class="each_item">
            <span>
              <p class="category">Groceries</p>
              <p class="title">GS25</p>
            </span>
            <span>
              <p class="price_withdraw">-3000</p>
              <button class="delete_button">X</button>
            </span>
          </li>
```
리스트들이다보니 ul - li 태그를 쓰는건 어떨까용?